flask_jwt_extended/view_decorators.py

```diff
@@ -86,8 +86,9 @@ def verify_jwt_in_request(
         return None
 
     # Should be impossible to hit, this makes mypy checks happy
-    if not _request_ctx_stack.top:  # pragma: no cover
-        raise RuntimeError("No _request_ctx_stack.top present, aborting")
+    # TODO: Convert this check to the new `flask.g` object
+    # if not _request_ctx_stack.top:  # pragma: no cover
+    #     raise RuntimeError("No _request_ctx_stack.top present, aborting")
```

Comment:
This can be deleted now I think. Only reason this code existed was to satisfy mypy